src/pybind/mgr/dashboard/frontend/cypress/fixtures/orchestrator/inventory.json

```diff
@@ -0,0 +1,390 @@
+[
```

Comment:
What I don't understand is: why do you need this inventory fixture? should'n this be generated by cephadm?

Reply:
It's there for helping frontend developers reading the code or writing tests.